src/types/H5P/H5PBehaviour.ts

```diff
@@ -1,7 +1,7 @@
-import { H5PFieldGroup } from "./H5PField";
+import { H5PFieldGroup } from './H5PField';
```

Comment:
```suggestion
import { H5PFieldGroup } from 'h5p-types';
```

Reply:
H5P-types and H5P-utils will be moved to a new PR